ciao-controller/main.go

```diff
@@ -136,6 +136,8 @@ func main() {
 	identityURL = clusterConfig.Configure.IdentityService.URL
 	serviceUser = clusterConfig.Configure.Controller.IdentityUser
 	servicePassword = clusterConfig.Configure.Controller.IdentityPassword
+	keyringPath = clusterConfig.Configure.Storage.SecretPath
```

Comment:
Maybe we should only perform the assignment if the SecretPath is "".  Otherwise there's no point in providing defaults, as they'll never be used.